site/_posts/2015-01-20-jekyll-meet-and-greet.markdown

```diff
@@ -0,0 +1,20 @@
+---
+layout: news_item
+title: "Jekyll Meet & Greet at GitHub HQ"
+date: "2015-01-20 19:23:12 -0800"
+author: parkr
+categories: [meetup]
+---
+
+Hey! Our friends at GitHub have agreed to host a Jekyll meet & greet on
+**February 5, 2015 at 7pm**. The event will be hosted at [GitHub's
+Headquarters](https://www.google.com/maps/place/GitHub/@37.782084,-122.391403)
```

Comment:
Correct me if I'm wrong, but according to [various GitHub meetup posts](https://github.com/blog/search?q=Colin+P+Kelly) and [a Google Street View image featuring your awesome Octocat sign](https://www.google.com/maps/@37.78235,-122.391131,3a,50.7y,225.92h,86.66t/data=!3m4!1e1!3m2!1sLHQUzmqQ3GAE2u1JzPBWmg!2e0), is GitHub's address 88 Colin P. Kelly (not 85)?